LiteFarmOrg/LiteFarm
# Add time offsets to farm creation in backend

**To Test:**
1. You would need to review this [PR](https://github.com/LiteFarmOrg/LiteFarm/pull/1944) first and follow the steps there.
2. Create a new farm in multiple locations
3. Check their UTC offset in the database `farm` table (should be in seconds)
